amp.php

```diff
@@ -161,6 +161,22 @@
 	);
 }
 
+// DEV_CODE. This block of code is removed during the build process.
+if ( file_exists( AMP__DIR__ . '/amp-beta-tester.php' ) ) {
+	add_filter(
+		'site_transient_update_plugins',
+		function ( $updates ) {
```

Comment:
No `$this` context, so closure can be static for performance reasons.
```suggestion
		static function ( $updates ) {
```